packages/frontend/src/utils/assets.ts

```diff
@@ -44,6 +44,14 @@ function assetHashToInfo(
   assetDetails?: AssetDetails
 ): AssetInfo {
   if (!assetDetails) {
+    if (assetHash.startsWith('0x04')) {
+      return {
+        name: 'Mintable NFT',
+        symbol: `Mintable # ${assetHash.substring(4)}`,
+        isUnknownHash: true,
+        imageUrl: '/images/unknown-asset.svg',
+      }
```

Comment:
Why do we need that?

Reply:
It's a temporary solution for Phase1 - we don't have any info on Mintable NFTs, so on frontend only we display a placeholder.